.github/workflows/build_documentation.yml

```
@@ -84,7 +84,7 @@ jobs:
           cd doc-builder
           if [[ `git status --porcelain` ]]; then
             git add build
-            git commit -m "Updated with commit ${{ github.sha }}"
+            git commit -m "Updated with commit https://github.com/huggingface/transformers/commit/${{ github.sha }}"
```

```suggestion
            git commit -m "Updated with commit ${{ github.sha }} \n\nSee: https://github.com/huggingface/transformers/commit/${{ github.sha }}"
```

git has both a commit title and a description and i think it's the description that would be auto-linked

---

Ah ok, I misunderstood what you wanted.